lib/youtube-dl.js

```diff
@@ -45,15 +45,30 @@ var ytdl = module.exports = function(videoUrl, args, options) {
 
     var item = (!data.length) ? data : data.shift();
 
+    /**
+     * Fix for pause/resume downloads
+     */
+    var headers = {
+      'Host': url.parse(item.url).hostname
+    };
+
+    if (typeof options.start !== 'undefined' && options.start > 0) {
```

Comment:
`options` should be check if its defined, or set to `{}` above if undefined. and you only need the 2nd check, the `options.start > 0` since it will be false if `options.start` is undefined.